crates/lib3h_protocol/src/protocol.rs

```diff
@@ -58,10 +58,14 @@ pub enum ClientToLib3hResponse {
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub enum Lib3hToClient {
     // -- Connection -- //
+    /// Notification of successful connection to a network
+    Connected(ConnectedData),
     /// Notification of disconnection from a network
     Disconnected(DisconnectedData),
 
     // -- Direct Messaging -- //
+    /// the response received from a previous `SendDirectMessage`
+    SendDirectMessageResult(DirectMessageData),
```

Comment:
This is a response to `ClientToLib3h::SendDirectMessage`, we don't need to add it here, it's already `ClientToLib3hResponse::SendDirectMessageResult`

Reply:
Its used for doing a publish() so it can't be in the Response enum.
We can't do a respond() to a ClientToLib3h::SendDirectMessage since there is no response at the transport protocol layer.

Reply:
Right! yes, I got this one backwards, sorry : )